boards/arm/96b_aerocore2/96b_aerocore2_defconfig

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ARM=y
```

Comment:
ARM is now deduced from SoC selection.
Setting it in board _defconfig generates an error (this is what is screwing our shippable results)